src/RoadRegistry.BackOffice.ZipArchiveWriters/ExtractHost/RoadSegmentNumberedRoadAttributesToZipArchiveWriter.cs

```diff
@@ -29,10 +30,7 @@ public class RoadSegmentNumberedRoadAttributesToZipArchiveWriter : IZipArchiveWr
         if (request == null) throw new ArgumentNullException(nameof(request));
         if (context == null) throw new ArgumentNullException(nameof(context));
 
-        var attributes = await context.RoadSegmentNumberedRoadAttributes
-            .InsideContour(request.Contour)
-            .ToListAsync(cancellationToken);
-        var dbfEntry = archive.CreateEntry("eAttGenumweg.dbf");
+        var attributes = await context.RoadSegmentNumberedRoadAttributes.ToListWithPolygonials(request.Contour, (dbSet, polygon) => dbSet.InsideContour(polygon), x => x.Id, cancellationToken); var dbfEntry = archive.CreateEntry("eAttGenumweg.dbf");
```

Comment:
use a line-separated method chain (like what it was)

Reply:
Done